docs/source/tutorial_interpolation.md

```diff
@@ -0,0 +1,464 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.15.2
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+```{try_on_binder}
+```
+
+```{code-cell} ipython3
+:load: myst_code_init.py
+:tags: [remove-cell]
+
+
+```
+
+# Tutorial: Interpolation-based methods for LTI systems
+
+Here we discuss interpolation-based methods
+(also known as:
+moment matching,
+Krylov subspace methods, and
+Padé approximation)
+for LTI systems.
+We start with simpler approaches
+(higher-order interpolation at single point {cite}`G97`) and
+then move on to bitangential Hermite interpolation
+which is directly supported in pyMOR.
+
```

Comment:
It would possibly be helpful to write a bit more motivation for interpolatory model reduction here or in the section below. Transfer functions are known from the LTI systems tutorial. By simply pointing out the relationship between the input-output mapping which we are interested in and the transfer function it will hopefully immediately make sense why we should even care about these interpolatory methods.